### DISCUSSION TITLE
How to run the binary file and set configurations?

### DISCUSSION THREAD

**Discussion by reactivetype** [mudler/LocalAI#1240] [Q&A]
I have downloaded the binary from releases and looked at the help to see the necessary flags. t an example, it's very hard to guess how to run this binary file.
How do I define the configuration that --config-file should point to? For newbies, this is not obvious from the documentation. 
Also, how do I specify which model in /models/ folder I want to use? 
Could you please provide some clear example here?  Thank you.

**Comment by lunamidori5** [comment]
> I have downloaded the binary from releases and looked at the help to see the necessary flags. t an example, it's very hard to guess how to run this binary file. How do I define the configuration that --config-file should point to? For newbies, this is not obvious from the documentation. Also, how do I specify which model in /models/ folder I want to use? Could you please provide some clear example here? Thank you.

@reactivetype https://localai.io/basics/build/
